### PR TITLE
Update theme colors

### PR DESCRIPTION
**Pull request by irfano**
Closes: woocommerce/woomobile-private#381
Closes: woocommerce/woomobile-private#402

### Description

This PR updates the Woo purple colors of the app according to the v4.0.0 of https://color-studio.blog/. 
No test was added since this only changes colors.

### Steps to reproduce

Just run the app and view as many screens as possible.

### Testing information

Please also test in the dark mode to check for any contrast issues. Design file to compare: Ntkk0ER4hHt5y9KjEGIiGk-fi-2481_1556

### The tests that have been performed

Viewed some screens.

### Images/gif

splash
my-store
orders
products
reviews
reviews-dark
payments
collect-payment

- [x] I have considered if this change warrants release notes and have added them to `RELEASE-NOTES.txt` if necessary. Use the "[Internal]" label for non-user-facing changes.

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:

- [ ] The PR is small and has a clear, single focus, or a valid explanation is provided in the description. If needed, please request to split it into smaller PRs.
- [ ] Ensure Adequate Unit Test Coverage: The changes are reasonably covered by unit tests or an explanation is provided in the PR description.
- [ ] Manual Testing: The author listed all the tests they ran, including smoke tests when needed (e.g., for refactorings). The reviewer confirmed that the PR works as expected on big (tablet) and small (phone) in case of UI changes, and no regressions are added.

### CODE REVIEW

**Review by hafizrahman — APPROVED** [review]
The current code changes look good so I'm pre-approving. Please check my next comment for my findings and areas where we might want to improve.

**Comment by hafizrahman** [comment]
| Product filter > Product type | Product details: dark | Product details: light |
|-|-|-|
|  |  |  |
| Explore text maybe should be white? | "Write with AI" purple on gray is hard to read. Maybe should be white? | "Got it" purple on gray is hard to read. Maybe should be white? |

**Comment by irfano** [comment]
Thanks, Hafiz! Good feedback! 

> Explore text maybe should be white?

Good catch! I think it was a bug since the button didn’t have style. I’ve updated it.
Your other comments about the purple button on a dark background are part of a broader topic, as we use it on many screens. I’ll share your feedback with Wagner.

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/common/composeui/WooPosTheme.kt [hunk 1]
```diff
@@ -178,7 +150,7 @@ private val DarkColorPalette = darkColors(
 )
 
 private val LightColorPalette = lightColors(
-    primary = WooPosColors.Purple50,
+    primary = WooPosColors.WooPurple50,
```
Comment (hichamboushaba) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/common/composeui/WooPosTheme.kt:153]: @irfano I see that you kept the pruple50 for the POS theme, I didn't do any updates yet in iOS, so I wanted to ask if we should confirm with design whether we should change the POS them to use purple40 too or to keep it like this, WDYT?

**Comment by wagveloso** [comment]
Hi there! I think we can work on finding a good solution.

The purple graphics, like the CTA, pass on the contrast check for both, dark gray and black backgounds. However, the text needs to be white on them.

For links, normal text on dark backgrounds, my recommendation is to go with white, because the purple is not creating a good contrast and not passing on the contrast checker. However, I believe that changing all the links to white will add a lot of work for you guys.

What you think we should do?

**Comment by irfano** [comment]
>However, I believe that changing all the links to white will add a lot of work for you guys.

@wagveloso, yes, it’s a big task. If we change all the purple text to white, it will make it harder to differentiate certain elements, such as headers and list items, in some cases. So, I think this would be a significant change for the app.



We currently use **Purple 30** for dark mode. What if we apply the same for Woo 2.0? I’ll share some examples. Do you think it improves contrast?

|Purple 40|Purple 30|
|-|-|
|||
|||

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/common/composeui/WooPosTheme.kt (hunk 1 above)
Comment (irfano) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/common/composeui/WooPosTheme.kt:153]: I asked to see if there’s any reason to use a different theme for WooPos, but it looks like there isn’t. (p1734456791340939/1734454071.869999-slack-C025A8VV728) So, I’ll update the primary color to Purple 40.

**Comment by wagveloso** [comment]
Thanks for sharing the screens! Purple 30 looks much better @irfano . I vote for using Purple 30.

**Comment by irfano** [comment]
I agree! It looks better. Then we’ll update it to Purple 30 for dark mode.

**Pull request merged**